src/m365/planner/commands/bucket/bucket-remove.ts

```diff
@@ -0,0 +1,255 @@
+import { Cli, Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import Utils from '../../../../Utils';
+import { GraphItemsListCommand } from '../../../base/GraphItemsListCommand';
+import { Bucket } from '../../Bucket';
+import commands from '../../commands';
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  bucketId: string;
+  bucketName: string;
```

Comment:
Since `bucketName` is optional, let's denote is as such using `bucketName?: string`